src/utils/prefs.js

```diff
@@ -55,6 +55,7 @@ if (isDevelopment()) {
   pref("devtools.debugger.features.pause-points", true);
   pref("devtools.debugger.features.component-pane", false);
   pref("devtools.debugger.features.skip-pausing", false);
+  pref("devtools.debugger.features.autocomplete-expressions", false);
```

Comment:
this will also need to be in `assets/panel/prefs` devtools.debugger.features.autocomplete-expressions